xla/service/service_executable_run_options.h

```diff
@@ -28,14 +28,18 @@ namespace xla {
 class ServiceExecutableRunOptions {
  public:
   using StreamBorrower = std::function<StatusOr<StreamPool::Ptr>(int)>;
+  using StreamBorrowerWithPriority = std::function<StatusOr<StreamPool::Ptr>(
+      int, stream_executor::StreamPriority)>;
```

Comment:
Again, any possibility to unify by just adding a default valued parameter? Or since this is a std::function may be we just need to update all the callers, wight might be ok?